common/src/main/resources/messages/messages.properties

```diff
@@ -588,3 +588,25 @@ aws.variant.migration.successful=Successfully migrated the cloud formation templ
 aws.variant.migration.failed=Failed to migrate the cloud formation template
 
 authorization.doc=Please read documentation about user management in official doc: https://docs.cloudera.com/management-console/cloud/user-management/topics/mc-managing-user-access.html. More specifically, you can read about roles here: https://docs.cloudera.com/management-console/cloud/user-management/topics/mc-understanding-roles-resource-roles.html. If you need more detailed information about permissions, you need to check underlying service of user management.
+
+conclusion.node.status.monitor.unreachable=Node status monitor is unreachable. The possible reasons are: the cluster doesnt support the node status monitor service, or the network configuration is not CDP compatible, or some required components are in bad health. Please ssh into the cluster and run cdp doctor.
+conclusion.node.status.monitor.failed=There are unhealthy services on nodes: {0}. Please check the instances on your cloud provider for further details.
+conclusion.node.status.monitor.failed.details=There are unhealthy services on nodes: {0}
+conclusion.salt.collect.unreachable.found=Unreachable nodes: {0}. We detected that cluster members can?t communicate with each other. Please validate if all cluster members are available and healthy through your cloud provider.
+conclusion.salt.collect.unreachable.found.details=Unreachable salt minions based on salt ping: {0}
+conclusion.salt.collect.unreachable.failed=Collecting unreachable nodes failed. Please validate if all cluster members are available and healthy through your cloud provider.
```

Comment:
"through your cloud provider" maybe "on your cloud provider" is better, we use that phrase everywhere else